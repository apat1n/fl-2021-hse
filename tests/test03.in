(a, b, q), (q_1, q_2), q_1, f(q_2), [(q_1, q_2, (a, b), (q_1, q_1, (q))]