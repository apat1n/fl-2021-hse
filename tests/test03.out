left-parenthesis ( 1 0
ERROR in line 1, column 1, symbol a
